package.json

```diff
@@ -54,6 +54,7 @@
     "@nivo/core": "0.79.0",
     "@nivo/line": "0.79.0",
     "@nivo/scales": "0.79.0",
+    "@tanstack/react-virtual": "^3.0.0-beta.49",
```

Comment:
Could we maybe avoid a beta version? 🙏

Reply:
For posterity: this is the same version we use in brain-app, and we ran into issues attempting to upgrade a while back so safer to leave pinned for now.